packages/vue/examples/with-nuxt/nuxt.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
 	/*
 	 ** Plugins to load before mounting the App
 	 */
-	plugins: ['~/plugins/reactivesearch-vue'],
+	plugins: [{ src: '~/plugins/reactivesearch-vue', ssr: false }],
```

Comment:
It doesn't solve the issue, it means that you're using ReactiveSearch without SSR.
We have to support SSR.

Reply:
https://github.com/NightCatSama/vue-slider-component/issues/114#issuecomment-353827607

Reply:
I already know that but it is not gonna work.
Try some other thing, if doesn't work then don't upgrade the `vue-slider-component`.

Reply:
@ShahAnuj2610 You can try it once https://nightcatsama.github.io/vue-slider-component/#/?hash=server-side-rendering-ssr